File: c#/customer.cs
using System;
public class Customer{
	protected string name;
	protected int age;
    public string food;
    public Customer(string _name, int _age,string _food) {
	    name = _name;
	    age = _age;
        food = _food;
    }

    ~Customer() {
        System.Console.WriteLine("Instance removed");
    }
    public void order() {
        System.Console.WriteLine(name+"  order");
        
    }
    public void pay() {
	    System.Console.WriteLine(name+"  pay");
        List<string> foodlist;
        foodlist = food.Split(',').ToList();
        for (int i=0; i < foodlist.Count; i++){
            var a = foodlist[i];
            Owner.plus(Menu.returnprice(a));
        }
    }
}

File: c++/customer.cpp
#include "customer.hpp"
#include <iostream>

using namespace std;

Customer::Customer(int _cus_id){
    cus_id = _cus_id;
}

Customer::~Customer(){
    cout << "Instance removed." << endl;
}

void Customer::order_food(){
    cout << cus_id << "order food to server"<<endl;
}

void Customer::pay_food(){
    cout << cus_id << "pay food to server" <<endl;
}